src/protocols/rdp/rdp_settings.c

```diff
@@ -854,6 +854,10 @@ guac_rdp_settings* guac_rdp_parse_args(guac_user* user,
     settings->timezone =
         guac_user_parse_args_string(user, GUAC_RDP_CLIENT_ARGS, argv,
                 IDX_TIMEZONE, NULL);
+    
+    /* If no timezone explicitly set, try to pull from tunnel */
```

Comment:
guacd doesn't know about the tunnel. ;)

This is good to note, but will need to be noted differently to be correct from the perspective of this part of the stack.

Reply:
Removed comment, as the section got removed and used default value.